textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/queryrewriter/QueryRewriterTest.java

```diff
@@ -1,6 +1,10 @@
 package edu.uci.ics.textdb.dataflow.queryrewriter;
 
+import edu.uci.ics.textdb.api.common.IField;
```

Comment:
@rajesh9625 The function definition of `containsAllResults()` allows for only lists of the type `ITuple`. I cannot pass two lists of strings to it.